tango/__main__.py

```diff
@@ -331,9 +335,62 @@ def run(
         step_name=step_name,
         name=name,
         called_by_executor=_CALLED_BY_EXECUTOR,
+        allow_dirty=allow_dirty,
     )
 
 
+@main.command(hidden=True)
+@click.argument(
+    "experiment",
+    type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+)
+@click.argument(
+    "step_name",
+    type=str,
+)
+@click.argument(
+    "workspace_url",
+    type=str,
+)
+@click.option(
+    "-i",
+    "--include-package",
+    type=str,
+    help="Python packages or modules to import for tango components.",
+    multiple=True,
+)
+def beaker_executor_run(
+    experiment: str,
+    step_name: str,
+    workspace_url: str,
+    include_package: Optional[Sequence[str]] = None,
+):
+    """
+    This command is only used internally by the BeakerExecutor.
+    """
+    from tango.common.logging import do_json_logging
+    from tango.executor import Executor
+
+    if include_package:
+        for package_name in include_package:
+            import_extra_module(package_name)
+
+    # Load step graph and step.
+    step_graph = StepGraph.from_file(experiment)
+    step = step_graph[step_name]
+
+    # Initialize workspace and executor.
+    workspace = Workspace.from_url(workspace_url)
+    executor = Executor(workspace=workspace, include_package=include_package)
```

Comment:
So the `BeakerExecutor` uses the default executor when running individual steps?

Reply:
Yes, that's worth a comment. I'll add one.